File: test_upload.py
import urllib.request
import json

url = 'https://script.google.com/macros/s/AKfycbxGO8qhJGBMmDueIkz-lse9c3PKsr7lGDdItToojUi-zUozIl6ogt-J-KmGkxKlzbe1Eg/exec'
data = {
  'action': 'uploadFile',
  'base64': 'data:text/plain;base64,SGVsbG8gV29ybGQh',
  'mimeType': 'text/plain',
  'filename': 'test.txt'
}

req = urllib.request.Request(url, data=json.dumps(data).encode('utf-8'), method='POST', headers={'Content-Type': 'text/plain'})
try:
    with urllib.request.urlopen(req) as response:
        print('Response:', response.read().decode('utf-8'))
except Exception as e:
    print('Error:', e)

File: test_upload.js
const fetch = require('node-fetch');
const url = 'https://script.google.com/macros/s/AKfycbxGO8qhJGBMmDueIkz-lse9c3PKsr7lGDdItToojUi-zUozIl6ogt-J-KmGkxKlzbe1Eg/exec';
const base64Str = 'data:text/plain;base64,SGVsbG8gV29ybGQh'; // "Hello World!"
const data = {
  action: 'uploadFile',
  base64: base64Str,
  mimeType: 'text/plain',
  filename: 'test.txt'
};

fetch(url, {
  method: 'POST',
  body: JSON.stringify(data),
  headers: { 'Content-Type': 'text/plain' }
})
.then(res => res.json())
.then(data => console.log('Response:', data))
.catch(err => console.error('Error:', err));
